app/views/movies/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Movie <%= @movie.title %> Info</h1>
+
+<p>Movie Title: <%= @movie.title %></p>
+<p>Movie Creation Year: <%= @movie.creation_year %></p>
+<p>Movie Genre: <%= @movie.genre %></p>
+
+<p>Actors:</p>
+<% @movie.actors_list.each do |actor| %>
+<p><%= actor.name %></p>
+<% end %>
+
+<p>Average Actors Age: <%= @movie.ave_actors_age %></p>
+
+
+<%= form_with url: "/movies/#{@movie.id}/actors", method: :post, data: {turbo:false},local: true do |form| %>
+  
+  <%= form.label :actor_id, "Actor ID" %>
+  <%= form.text_field :actor_id %>
+
+  <%= form.submit "Submit" %>
+<% end %>
```

Comment:
Yes! This form is perfect and follows restful conventions perfectly! You're using the correct verb (POST) and sending the data to the correct path, great job Sooyung!